### PR TITLE
go: update to 1.11.2

### PR DESCRIPTION
**Pull request by bobertlo**

### CODE REVIEW

File: srcpkgs/go/template [hunk 1]
```diff
@@ -10,7 +10,7 @@ maintainer="Michael Aldridge <maldridge@VoidLinux.eu>"
 homepage="http://golang.org/"
 license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/go/template]: BSD-3-Clause

Comment (bobertlo) [reply srcpkgs/go/template]: What is the issue? The LICENSE is installed with the package.

Comment (bobertlo) [reply srcpkgs/go/template]: Or is it just the formatting?

Comment (maxice8) [reply srcpkgs/go/template]: https://spdx.org/licenses/BSD-3-Clause

**Comment by maxice8** [comment]
https://github.com/void-linux/void-packages/commit/47ea14468d09db339ef17a2ebfdc188127b99309

**Pull request closed without merging**